src/peft/utils/save_and_load.py

```diff
@@ -506,13 +506,13 @@ def get_hub_filename(use_safetensors=True):
         # if in offline mode, check if we can find the adapter file locally
         hub_filename = get_hub_filename(use_safetensors=True)
         try:
-            filename = hf_hub_download(model_id, hub_filename, local_files_only=True)
+            filename = hf_hub_download(model_id, hub_filename, local_files_only=True, **hf_hub_download_kwargs)
```

Comment:
I suspect this line will fail if `hf_hub_download_kwargs` contains a key `local_files_only` which is the case if someone does `AutoModelForCausalLM.from_pretrained(model_id, local_files_only=True)`.